### PR TITLE
[DO_NOT_MERGE] [NP-4451] Beneficial owner title needs to be translated

### PR DESCRIPTION
**Pull request by Mykola97**
https://nanopay.atlassian.net/browse/NP-4451

### CODE REVIEW

**Comment by jlhughes** [comment]
Are we adding another property just to fix BeneficialOwners view?

File: src/foam/u2/view/TitledArrayView.js [hunk 1]
```diff
@@ -27,6 +27,10 @@ foam.CLASS({
     {
       name: 'title',
       documentation: 'This property is used to populate the header for each individual array element'
+    },
```
Comment (kgrgreer) [inline src/foam/u2/view/TitledArrayView.js:30]: How is this different than 'name'?

**Comment by Mykola97** [comment]
> Are we adding another property just to fix BeneficialOwners view?

Yes

**Comment by jlhughes** [comment]
K, don't like this.  Let's talk at next standup.

**Review by sarthak-marwaha — APPROVED** [review]

**Pull request merged**